nservicebus/azure-service-bus/keep-connectionstring-safe.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Keep connection string safe
+summary: Configuring Azure ServiceBus transport to secure connection strings. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+- Security
+reviewed: 2016-04-26
+---
+
+include: keep-connection-string-safe
+
+To enable this behavior, apply the following configuration:
```

Comment:
With snippet like that each time we add something, we have to keep in mind that it's going to be used in multiple places. Is that kind of re-use is desired? I don't think this "duplication" is bad. Plus, the ability of showing what would be rendered in version 6 w/o safe connection strings would be hurt since for ASB and ASQ it's not the same connection string type. My 2c :)